tests/printer/comments/expected/pattern.res.txt

```diff
@@ -59,7 +59,19 @@ let /* before */ {
 
 // Ppat_or
 let /* b1 */ Blue /* b2 */ | /* c1 */ Red /* c2 */ = color
-let /* b1 */ Blue /* b2 */ | /* c1 */ Red | /* d1 */ Green /* d2 */ = color
+let /* b1 */ Blue /* b2 */ | /* c1 */ Red /* c2 */ /* d1 */ | Green /* d2 */ = color
```

Comment:
@IwanKaramazow Input here was

```rescript
let /* b1 */ Blue /* b2 */ | /* c1 */ Red /* c2 */ | /* d1 */ Green /* d2 */ = color
```

I think we cannot prevent c2 and d1 from moving together, as they are between the two patterns.

Reply:
This seems like a regression, the `walkList` helper will take care of the correct behaviour.